README.md

```diff
@@ -114,6 +114,17 @@ const getUserInfoList = async () => {
 }
 ```
 
+### Authenticate using Azure RBAC
```

Comment:
I would rather the flow of the application take care of all the settings and not require the user to take manual action.  

We are experimenting with adding an AZURE_AUTH_TYPE parameter that can get set to rbac or keys and the bicep does the appropriate thing based on that.

It's not finalized, but should be soon.